ui/app/components/console/ui-panel.js

```diff
@@ -8,22 +9,38 @@ import {
   executeUICommand,
 } from 'vault/lib/console-helpers';
 
-const { inject, computed } = Ember;
+const { inject, computed, getOwner, run } = Ember;
 
 export default Ember.Component.extend({
   classNames: 'console-ui-panel-scroller',
   classNameBindings: ['isFullscreen:fullscreen'],
   isFullscreen: false,
   console: inject.service(),
+  router: inject.service(),
   inputValue: null,
   log: computed.alias('console.log'),
 
+  didReceiveAttrs() {
+    let val = this.get('inputValue');
+    let oldVal = this.get('oldInputValue');
+    this.set('valChanged', val !== oldVal);
+    this.set('oldInputValue', val);
+  },
+
+  didRender() {
+    if (this.get('valChanged')) {
+      // make sure we're scrolled to the bottom;
+      this.scrollToBottom();
+    }
+  },
```

Comment:
Instead of using the `valChanged` attribute, could this be achieved with a `run.schedule('afterRender', () => this.scrollToBottom())`? Just saves having an extra explicit state attribute, not really any other advantage.